res/css/components/views/location/_ShareType.scss

```diff
@@ -24,6 +24,36 @@ limitations under the License.
     padding: 60px $spacing-12 $spacing-32;
 
     color: $primary-content;
+
+    .mx_ShareType_wrapper_options {
+        display: flex;
+        flex-flow: column;
+        row-gap: $spacing-12;
+        width: 100%;
+        margin-top: $spacing-12;
+
+        .mx_ShareType_option {
+            display: flex;
+            flex-direction: row;
```

Comment:
This is the default value. So I think we can remove it.

(same applies to `jusitfy-content`)

Reply:
Exactly, I'm removing it soon.